src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+/**
+ * Class describe the tractor equipped with a hinged dump.
+ */
+public class Bulldozer extends Machine {
+    //This is the template for the first part of the message.
```

Comment:
let's remove all redundant comments from your code
your code should be understandable without comment